webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require("path");
+const HTmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = (env) => {
+  return {
+    mode: env.WEBPACK_SERVE ? "development" : "production",
+
+    entry: "./src/index.js",
+    output: {
+      filename: "bundle.js",
+      path: path.join(__dirname, "built"),
```

Comment:
Seems like you have a typo here. Probably you meant `build` instead of `built`

Reply:
fixed